static/scripts/helpers/imageNotFound.js

```diff
@@ -0,0 +1,13 @@
+const imageNotFound = () => {
+	const images = $('.ck.ck-content.ck-editor__editable').find('img');
+	for (const image of images) {
+		$(image).on('error', function () {
```

Comment:
Den Callback könnte man noch in ne eigene Funktion auslagern. addOnErrorAttributeWithFallbackImage